Dockerfile

```diff
@@ -4,10 +4,11 @@ COPY . /go/src/github.com/cloudflare/unsee
 
 ARG VERSION
 
-RUN go install \
-    -ldflags "-X main.version=${VERSION:-dev}" \
-    github.com/cloudflare/unsee && \
-    rm -fr /go/src
+RUN apk add --update make git nodejs
```

Comment:
Either run this as one line (to avoid extra layers), or use Docker's [multi-stage builds](https://docs.docker.com/engine/userguide/eng-image/multistage-build/)